src/libs/utils.js

```diff
@@ -150,6 +151,34 @@ export const NavigationUtils = {
   }
 };
 
+//get information on datasets, consent, researcher, and access request
+export const getDarData = async (darId) => {
+  let datasets;
+  let darInfo;
+  let consent;
+  let researcherProfile;
+
+  try {
+    darInfo = await DAR.getPartialDarRequest(darId);
+    const researcherPromise = await Researcher.getResearcherProfile(darInfo.userId);
+    const datasetsPromise = darInfo.datasetIds.map((id) => {
+      return DataSet.getDataSetsByDatasetId(id);
+    });
+    const consentPromise = await DAR.getDarConsent(darId);
+    [consent, datasets, researcherProfile] = await Promise.all([
+      consentPromise,
+      Promise.all(datasetsPromise),
+      researcherPromise
+    ]);
+
+  } catch (error) {
+    Notifications.showError({text: 'Error initializing DAR Data'});
```

Comment:
I know this is not your code originally, but the error message seems unhelpful, especially with the acronym. I think this might be more helpful to a user:
```suggestion
    Notifications.showError({text: 'Error retrieving Data Access Request information, please contact support.'});
```